apps/passport-server/src/routing/routes/telegramRoutes.ts

```diff
@@ -125,4 +125,29 @@ export function initTelegramRoutes(
       res.status(500).sendFile(path.resolve("resources/telegram/error.html"));
     }
   });
+
+  app.get("/telegram/anon", async (req: Request, res: Response) => {
+    try {
```

Comment:
we should throw an error if telegramService is not defined

Reply:
Done